tests/pkg_littlefs2/Makefile

```diff
@@ -0,0 +1,32 @@
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove \
+                             arduino-leonardo \
+                             arduino-nano \
+                             arduino-uno \
+                             i-nucleo-lrwan1 \
+                             nucleo-f031k6 \
+                             nucleo-f042k6 \
+                             nucleo-l031k6 \
+                             nucleo-l053r8 \
+                             nucleo-f030r8 \
+                             stm32f0discovery \
+                             stm32l0538-disco \
```

Comment:
So when you move this to `Makefile.ci` you can also add `waspmote-pro`, `atmega328p` and `stm32f030f4-demo`.